### PR TITLE
Category selection (Week 3-4)

### PR DESCRIPTION
**Pull request by radek203**

### CODE REVIEW

File: appmain.py
Comment (radek203) [inline appmain.py:1]: to remove

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .idea
 __pycache__
-.env
\ No newline at end of file
+.env
+test_noapp.py
```
Comment (radek203) [inline .gitignore:4]: to remove

File: main.py
Comment (radek203) [inline main.py:1]: get_top_n_categories should be called e.g. after user (9) rates

File: data/categories_1.csv
Comment (radek203) [inline data/categories_1.csv:1]: We don't need additional file. To remove.

File: data/ratings.csv [hunk 2]
```diff
@@ -89,4 +89,4 @@ userId,categoryId,rating,tags
 1,3,10,stars|galaxy|spaceship|magic
 1,3,9,phoenix|magic|castle|wizard
 1,3,8,unicorn|fire|knight|forest
-1,1,10,waterfall|forest|trees|rainbow
+1,1,10,waterfall|forest|trees|rainbow
```
Comment (radek203) [inline data/ratings.csv:92]: Why is this in commit?

File: backend/callbacks.py
Comment (radek203) [inline backend/callbacks.py:1]: Edit this file to comply with required fixes.

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,2 @@
 .idea
 __pycache__
-.env
```
Comment (radek203) [inline .gitignore:3]: Revert it

File: data/ratings.csv [hunk 4]
```diff
@@ -89,4 +89,3 @@ userId,categoryId,rating,tags
 1,3,10,stars|galaxy|spaceship|magic
 1,3,9,phoenix|magic|castle|wizard
 1,3,8,unicorn|fire|knight|forest
-1,1,10,waterfall|forest|trees|rainbow
```
Comment (radek203) [inline data/ratings.csv:92]: Why?

File: main.py
Comment (radek203) [inline main.py:1]: Revert it

File: data/ratings.csv (hunk 4 above)
Comment (olastrz) [reply data/ratings.csv:92]: Cause last comments was why was it in commit and it was in there by mistake (not deleted after testing)

**Pull request closed without merging**